datamade/la-metro-dashboard
# Pull Friday and Saturday scrapes into combined DAG, use 24-hour time in windowed scrape handlers

## Description

See title. I made these changes so support scrapes have a more expressive name (and to reduce redundancy), and to ensure that regular windowed scrapes run until the support window starts.

## Testing instructions

- Run the app locally and confirm there are no exceptions. Turn on the `fast_full_scraping` DAG and confirm it runs as expected, according to whether we're within the support window (i.e., if it's not between 9 p.m. UTC Friday and 6 a.m. Saturday, no-op.)
- Check on the DAG runs Monday to confirm fast full scrapes ran as expected during the support window. (It's not support night this week, so failures can be remedied next week.)
